src/pages/summer/Login/LoginInput/LoginInput.scss

```
@@ -0,0 +1,34 @@
+@import '../../../../styles/common.scss';
+
+.Login-summer .login-container form {
```

오... 컴포넌트 안에 저는 다 클래스명을 줬는데 이런식으로 할 수 있내요!! 크게 배워갑니다!

---

```suggestion
.user-login {
```
혹시 간단히 클래스 셀렉터로 처리하지 않으신 이유가 있을까요?

---

> 혹시 간단히 클래스 셀렉터로 처리하지 않으신 이유가 있을까요?

아니요 셀프 리팩토링을 하면서도 신경쓰지 못했던 부분입니다. 감사합니다!